our-work/software/nmr-web-app.yml

```diff
@@ -0,0 +1,15 @@
+title: NMR Web Application 
```

Comment:
Let's put the full name of NMR in the title since I wouldn't expect users to be familiar with it

Reply:
Will change now, thank you!